app/models/concerns/with_qualifications.rb

```diff
@@ -60,12 +60,28 @@ def qualifications
       end
     end
 
+    def full_qualifications
```

Comment:
Is there point to the array?

Reply:
No point at all.

https://github.com/DFE-Digital/publish-teacher-training/pull/3148/commits/6efd1eb0475b0753f472aeaffb7e27be30622a72